ujin/programmers/day08/다트게임.js

```diff
@@ -0,0 +1,26 @@
+function solution(dartResult) {
+  const answer = [];
+  let score = 0;
```

Comment:
`else if`, `switch - case` 같은 조건문을 객체를 활용 (Lookup Table) 하여 반복되는 코드를 줄일 수 있습니다. (반복되는 코드 `else if (조건) {}`)
```suggestion
  let score = 0;
  const check = {
    'S':(score) => {answer.push(score)},
    'D':(score) => { answer.push(score ** 2)},
    'T':(score) => { answer.push(score ** 3)},
    '*':(score) => {
      answer[answer.length - 2] *= 2;
      answer[answer.length - 1] *= 2;
    },
    '#':(score) => { answer[answer.length - 1] *= -1},
	};
```